src/test/resources/inputs/basics/paths.sql

```diff
@@ -139,7 +139,7 @@ SELECT t.x[0 + 1] AS v FROM t;
 -- SELECT t.x[ABS(1)] AS v FROM t;
 
 --#[paths-sfw-16]
-SELECT t.array[0] AS v FROM t;
+SELECT t."array"[0] AS v FROM t;
```

Comment:
self-review: should use qualified identifier since `array` is a reserved keyword in PLK-1.x